docs/activity.md

```diff
@@ -0,0 +1,241 @@
+</p>
+
+
+## What is all this about?
+
+<img width="30%" alt="microbit-universal-hex logo" src="/microbit-bird-activity/img/bird-with-microbit.jpg" align="right">
+
+Your task is to think like a bird and programme the micro:bit to react to the world like a bird.
+
+The micro:bit can communicate wirelessly with other micro:bits using radio.
+There will be a secret micro:bit in the room broadcasting radio messages to
+all the birds indicating different events like "saying hello" o "there is a cat
+around".
+
+Apart from that, the micro:bit contains other sensors that your bird will react
+to, for example, birds don't like to be shaken!
+
+To programme the micro:bit with Python we use 
+[MicroPython (click here for more info)](https://microbit-carlos.github.io/microbit-bird-activity/micropython/)
+and brand new
+[Python Editor (click here for more info)](https://microbit-carlos.github.io/microbit-bird-activity/editor/).
+
+
+## First steps - Load the starter project into the editor
+
+Okay, so the first step is to open this link to the Python Editor.
+This link contains a flag to enable additional beta features:
+<a href="https://python.microbit.org/v/beta?flag=audioSoundEffect" target="_blank">https://python.microbit.org/v/beta?flag=audioSoundEffect</a>
+
+To start the activity we first need to load a project hex file into the editor.
+
+Right click on this link and select "Save Link As" to save the hex file to your
+laptop: [https://github.com/microbit-carlos/microbit-bird-activity/raw/main/src/bird/bin/flockbird.hex](https://github.com/microbit-carlos/microbit-bird-activity/raw/main/src/bird/bin/flockbird.hex)
+
+And then drag and drop the hex file to the Python Editor. The editor might
+warn you that this action will replace the project, you can click "Replace" to
+accept it.
+
+### So... What am I looking at?
+
+The loaded hex file contains two Python files, to see them let's go to "Project"
+section on the left sidebar:
+
+![Python Editor Projects](img/editor-project.png)
+
+In this view you can switch between the `main.py` file and the `bird.py`.
+
+- The `bird.py` module contains some functionality for the bird to be able to
+react to the world.
+- The `main.py` file is the main code we will be working on.
+
+Both files have some comments that should help clarify what some of the
+code is doing, but there is no need to familiarise yourself with these yet,
+as we'll work through the files as part of this activity.
+
+The sidebar also have other useful sections:
+- Reference: A place to explore the features of your micro:bit with Python
+- API: The micro:bit MicroPython API developer documentation
+
+<img width="45%" alt="microbit-universal-hex logo" src="/microbit-bird-activity/img/editor-reference.png">
+<img width="45%" alt="microbit-universal-hex logo" src="/microbit-bird-activity/img/editor-api.png">
+
+
+## Reacting to events
+
+You'll see at the top of `main.py` a function called `hello()` with a decorator
```

Comment:
Do you have an authoritative source to link to on decorators? Might be handy if there are beginner programmers about?

Reply:
Real Python has a great article on decorators (on most Python features really): https://realpython.com/primer-on-python-decorators/
But it is really long and not something a user could quickly grok and move on, I'll see if I can find something on google.